src/pooling.jl

```diff
@@ -166,7 +180,32 @@ function meanpool(x, k::NTuple{N, Integer}; pad=0, stride=k) where N
 end
 
 
-for pool in [:maxpool, :meanpool]
+"""
+    lpnormpool(x, p::Number, k::NTuple{N, Integer}; pad=0, stride=k)
+
+Perform Lp pool operation with value of the Lp norm `p` and window size `k` on input tensor `x`, also known as LPPool in pytorch.
+This pooling operator from [Learned-Norm Pooling for Deep Feedforward and Recurrent Neural Networks](https://arxiv.org/abs/1311.1780).
+
+* `x` and `k`: Usually, ndim(x) ∈ [3, 5], length(k) ∈ [1, 3], s.t. ndim(x) == length(k) + 2
+* `p` is restricted to (0, Inf). Out-of-range `p` leads to undefined behavior.
+* `pad`: See [`pad_zeros`](@ref) for details.
+* `stride`: Stride for each spatial axis. `k` as default if not present.
+
+For each element `x` in (k × k) window, lpnormpool computes `(∑ x^p)^(1 / p)` as output.
+
+* When p = 1, lpnormpool(x, p, k) ./ prod(k) ≈ meanpool(x, k)
+* When p = 2, lpnormpool(x, p, k).^2 ./ prod(k) ≈ meanpool(x.^2, k)
```

Comment:
These are code, and don't really need bullet points:
```suggestion
Thus `lpnormpool(x, 1, k) ./ prod(k) ≈ meanpool(x, k)` and 
`lpnormpool(x, 2, k).^2 ./ prod(k) ≈ meanpool(x.^2, k).
```

Reply:
OK, now it looks more specific and solid.